web/app/js/components/util/TapUtils.jsx

```diff
@@ -304,16 +308,37 @@ export const srcDstColumn = (d, resourceType, ResourceLink) => {
     labels = d.destinationLabels;
   }
 
+  let link = (
+    !_.isEmpty(labels[resourceType]) ?
+      resourceShortLink(resourceType, labels, ResourceLink) :
+      display.str
+  );
+
+  const linkFn = e => {
+    e.preventDefault();
+  };
+
   let baseContent = (
-    <div className="src-dst-name">
-      { !_.isEmpty(labels[resourceType]) ? resourceShortLink(resourceType, labels, ResourceLink) : display.str }
-    </div>
+    <Link to="#" onClick={linkFn}>
+      <OpenInNewIcon fontSize="small" />
+    </Link>
   );
 
   return (
-    <Popover
-      popoverContent={(popoverResourceTable(d, ResourceLink))}
-      baseContent={baseContent} />
+    <Grid
+      container
+      direction="row"
+      alignItems="center"
+      spacing="8">
```

Comment:
this needs to be `spacing={8}`  or else there'll be console errors:
```
Warning: Failed prop type: Invalid prop `spacing` of value `8` supplied to `Grid`, expected one of [0,8,16,24,32,40].
    in Grid (created by WithStyles(Grid))
    in WithStyles(Grid) (at TapUtils.jsx:328)
    in td (created by TableCell)
    in TableCell (created by WithStyles(TableCell))
    in WithStyles(TableCell) (at BaseTable.jsx:118)
    in tr (created by TableRow)
    in TableRow (created by WithStyles(TableRow))
    in WithStyles(TableRow) (at BaseTable.jsx:116)
    in tbody (created by TableBody)
    in TableBody (created by WithStyles(TableBody))
    in WithStyles(TableBody) (at BaseTable.jsx:111)
    in table (created by Table)
    in Table (created by WithStyles(Table))
    in WithStyles(Table) (at BaseTable.jsx:103)
    in div (created by Paper)
    in Paper (created by WithStyles(Paper))
    in WithStyles(Paper) (at BaseTable.jsx:102)
    in BaseTable (created by WithStyles(BaseTable))
    in WithStyles(BaseTable) (at TopEventTable.jsx:93)
    in TopEventTable (at AppContext.jsx:10)
    in Unknown (at TopModule.jsx:341)
    in TopModule (at AppContext.jsx:10)
    in Unknown (at ResourceDetail.jsx:238)
    in div (at ResourceDetail.jsx:237)
```

Reply:
nice catch! fixed.